input/elasticapm/internal/modeldecoder/generator/cmd/main.go

```diff
@@ -61,8 +61,8 @@ func generateV2() {
 	if err != nil {
 		panic(err)
 	}
-	generateCode(p, parsed, []string{"metadataRoot", "errorRoot", "metricsetRoot", "spanRoot", "transactionRoot", "logRoot"})
-	generateJSONSchema(p, pkg, parsed, []string{"metadata", "errorEvent", "metricset", "span", "transaction"})
+	generateCode(p, parsed, []string{"metadataRoot", "errorRoot", "metricsetRoot", "spanRoot", "transactionRoot", "logRoot", "telemetryRoot"})
+	generateJSONSchema(p, pkg, parsed, []string{"metadata", "errorEvent", "metricset", "span", "transaction", "telemetry"})
```

Comment:
@carsonip if you can add this to https://github.com/elastic/apm-data/pull/17 that would be awesome as it's a bit tricky to guess where and how to add root objects.

Reply:
ah sorry I missed this message. Adding root objects is slightly harder since it requires changing `processor.go`. Let's discuss in #17 whether it should go into the PR